### ISSUE TITLE
Missing debugger toolbar icons in Rider 2021.3

### ISSUE THREAD

**Issue by tjrobinson** [wallabyjs/public#2938]
### Issue description or question

In Rider 2021.3.x (and possibly other JetBrains IDEs), the debugging icons/controls while debugging aren't shown.

It should look like this:

![image](https://user-images.githubusercontent.com/925311/155710263-46ce4cbe-f305-4f11-b5ab-f8bc08b22ba2.png)

But I see this:

![image](https://user-images.githubusercontent.com/925311/155710297-40d9cf31-bb1a-4b2d-9af3-9b39dd239345.png)

Instead I have to use the keyboard shortcuts, or the Search Everywhere menu, or customise my menus to include these items:

![image](https://user-images.githubusercontent.com/925311/155710417-13c42977-14ee-4fd6-b99b-dbd42d272407.png)

### Wallaby diagnostics report

<details>
  <summary>Click to expand</summary>

```
{
  editorVersion: 'JetBrains Rider 2021.3.3',
  pluginVersion: '1.0.232',
  editorType: 'IntelliJ',
  osVersion: 'win32 10.0.19044',
  nodeVersion: 'v14.18.1',
  coreVersion: '1.0.1230',
  checksum: 'ZjI2ODdkNmMwOThhOGUyOGE3MzA2MDAxMjVkZTI1NjAsMTY3NDY5MTIwMDAwMCww',
  config: {
    diagnostics: {
      angular: {
        workspace: {
          '$schema': './node_modules/@angular/cli/lib/config/schema.json',
          version: 1,
          defaultProject: 'app',
          newProjectRoot: 'projects',
          projects: {
            app: {
              root: '',
              sourceRoot: 'src',
              projectType: 'application',
              prefix: 'app',
              schematics: {},
              architect: {
                build: {
                  builder: '@angular-devkit/build-angular:browser',
                  options: {
                    outputPath: 'www',
                    index: 'src/index.html',
                    main: 'src/main.ts',
                    polyfills: 'src/polyfills.ts',
                    tsConfig: 'tsconfig.app.json',
                    assets: [ { glob: '**/*', input: 'src/assets', output: 'assets' }, { glob: '**/*.svg', input: 'node_modules/ionicons/dist/ionicons/svg', output: './svg' } ],
                    styles: [ './node_modules/@angular/material/prebuilt-themes/indigo-pink.css', 'src/theme/variables.scss', 'src/global.scss' ],
                    scripts: [],
                    aot: false,
                    vendorChunk: true,
                    extractLicenses: false,
                    buildOptimizer: false,
                    sourceMap: true,
                    optimization: false,
                    namedChunks: true,
                    allowedCommonJsDependencies: [ 'immediate', 'uuid', 'crypto-js/sha256' ],
                    webWorkerTsConfig: 'tsconfig.worker.json'
                  },
                  configurations: {
                    production: {
                      fileReplacements: [ { replace: 'src/environments/environment.ts', with: 'src/environments/environment.prod.ts' } ],
                      optimization: true,
                      outputHashing: 'all',
                      sourceMap: false,
                      namedChunks: false,
                      aot: true,
                      extractLicenses: true,
                      vendorChunk: false,
                      buildOptimizer: true,
                      budgets: [ { type: 'initial', maximumWarning: '2mb', maximumError: '5mb' } ]
                    },
                    staging: {
                      fileReplacements: [ { replace: 'src/environments/environment.ts', with: 'src/environments/environment.staging.ts' } ],
                      optimization: true,
                      outputHashing: 'all',
                      sourceMap: false,
                      namedChunks: false,
                      aot: true,
                      extractLicenses: true,
                      vendorChunk: false,
                      buildOptimizer: true,
                      budgets: [ { type: 'initial', maximumWarning: '2mb', maximumError: '5mb' } ]
                    },
                    test: {
                      fileReplacements: [ { replace: 'src/environments/environment.ts', with: 'src/environments/environment.test.ts' } ],
                      optimization: true,
                      outputHashing: 'all',
                      sourceMap: false,
                      namedChunks: false,
                      aot: true,
                      extractLicenses: true,
                      vendorChunk: false,
                      buildOptimizer: true,
                      budgets: [ { type: 'initial', maximumWarning: '2mb', maximumError: '5mb' } ]
                    },
                    dev: {
                      fileReplacements: [ { replace: 'src/environments/environment.ts', with: 'src/environments/environment.dev.ts' } ],
                      optimization: true,
                      outputHashing: 'all',
                      sourceMap: false,
                      namedChunks: false,
                      aot: true,
                      extractLicenses: true,
                      vendorChunk: false,
                      buildOptimizer: true,
                      budgets: [ { type: 'initial', maximumWarning: '2mb', maximumError: '5mb' } ]
                    },
                    ci: { progress: false }
                  }
                },
                serve: {
                  builder: '@angular-devkit/build-angular:dev-server',
                  options: { browserTarget: 'app:build' },
                  configurations: { production: { browserTarget: 'app:build:production' }, ci: { progress: false } }
                },
                'extract-i18n': { builder: '@angular-devkit/build-angular:extract-i18n', options: { browserTarget: 'app:build' } },
                test: {
                  builder: '@angular-devkit/build-angular:karma',
                  options: {
                    main: 'src/test.ts',
                    polyfills: 'src/polyfills.ts',
                    tsConfig: 'tsconfig.spec.json',
                    karmaConfig: 'karma.conf.js',
                    styles: [ './node_modules/@angular/material/prebuilt-themes/indigo-pink.css' ],
                    scripts: [],
                    assets: [ { glob: 'favicon.ico', input: 'src/', output: '/' }, { glob: '**/*', input: 'src/assets', output: '/assets' } ],
                    webWorkerTsConfig: 'tsconfig.worker.json'
                  },
                  configurations: { ci: { progress: false, watch: false } }
                },
                lint: { builder: '@angular-eslint/builder:lint', options: { lintFilePatterns: [ 'src/**/*.ts', 'src/**/*.html' ] } },
                e2e: {
                  builder: '@angular-devkit/build-angular:protractor',
                  options: { protractorConfig: 'e2e/protractor.conf.js', devServerTarget: 'app:serve' },
                  configurations: { production: { devServerTarget: 'app:serve:production' }, ci: { devServerTarget: 'app:serve:ci' } }
                },
                'ionic-cordova-build': { builder: '@ionic/angular-toolkit:cordova-build', options: { browserTarget: 'app:build' }, configurations: { production: { browserTarget: 'app:build:production' } } },
                'ionic-cordova-serve': {
                  builder: '@ionic/angular-toolkit:cordova-serve',
                  options: { cordovaBuildTarget: 'app:ionic-cordova-build', devServerTarget: 'app:serve' },
                  configurations: { production: { cordovaBuildTarget: 'app:ionic-cordova-build:production', devServerTarget: 'app:serve:production' } }
                }
              }
            }
          },
          cli: { analytics: '72cdc6d9-51ff-4161-a527-6180ec0e403a', defaultCollection: '@ionic/angular-toolkit' },
          schematics: { '@ionic/angular-toolkit:component': { styleext: 'scss' }, '@ionic/angular-toolkit:page': { styleext: 'scss' } }
        },
        main: '// This file is required by karma.conf.js and loads recursively all the .spec and framework files\r\n' +
          '\r\n' +
          "import 'zone.js/dist/zone-testing';\r\n" +
          "import { getTestBed } from '@angular/core/testing';\r\n" +
          'import {\r\n' +
          '  BrowserDynamicTestingModule,\r\n' +
          '  platformBrowserDynamicTesting\r\n' +
          "} from '@angular/platform-browser-dynamic/testing';\r\n" +
          'import { ngMocks } from "ng-mocks";\r\n' +
          '\r\n' +
          'declare const require: {\r\n' +
          '  context(path: string, deep?: boolean, filter?: RegExp): {\r\n' +
          '    keys(): string[];\r\n' +
          '    <T>(id: string): T;\r\n' +
          '  };\r\n' +
          '};\r\n' +
          '\r\n' +
          '// All methods in mock declarations and providers\r\n' +
          '// will be automatically spied on their creation.\r\n' +
          '// https://ng-mocks.sudo.eu/extra/auto-spy\r\n' +
          "ngMocks.autoSpy('jasmine');\r\n" +
          '\r\n' +
          '// First, initialize the Angular testing environment.\r\n' +
          'getTestBed().initTestEnvironment(\r\n' +
          '  BrowserDynamicTestingModule,\r\n' +
          '  platformBrowserDynamicTesting()\r\n' +
          ');\r\n' +
          '// Then we find all the tests.\r\n' +
          '\r\n' +
          '// And load the modules.\r\n' +
          '\r\n'
      }
    },
    testFramework: { version: 'jasmine@3.10.0', configurator: 'jasmine@2.1.3', reporter: 'jasmine@2.1.3', starter: 'jasmine@2.1.3', autoDetected: true },
    env: { kind: 'chrome', type: 'browser', params: {}, viewportSize: { width: 800, height: 600 }, options: { width: 800, height: 600 }, bundle: true },
    files: [
      { pattern: 'src/**/*.spec.ts', ignore: true, trigger: true, load: true },
      { pattern: 'src/polyfills.ts', ignore: true, trigger: true, load: true },
      { pattern: 'src/test.ts', ignore: true, trigger: true, load: true },
      { pattern: 'src/main.ts', ignore: true, trigger: true, load: true },
      { pattern: 'src/index.html', ignore: true, trigger: true, load: true },
      { pattern: 'src/test.base.ts', ignore: true, trigger: true, load: true },
      { pattern: 'src/test.wallaby.ts', ignore: false, trigger: true, load: true, order: 1 },
      { pattern: 'src/**/*.+(ts|js)', load: false, ignore: false, trigger: true, order: 2 },
      { pattern: 'src/**/*.+(css|less|scss|sass|styl|html|json|svg)', instrument: false, load: false, ignore: false, trigger: true, order: 3 }
    ],
    tests: [ { pattern: 'src/**/*.spec.ts', load: false, ignore: false, trigger: true, test: true, order: 4 } ],
    filesWithNoCoverageCalculated: [],
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    addModifiedTestFileToExclusiveTestRun: true,
    compilers: {},
    preprocessors: {},
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    workers: { initial: 0, regular: 0, recycle: false },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    mapConsoleMessagesStackTrace: false,
    extensions: {},
    reportUnhandledPromises: false,
    throwOnBeforeUnload: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    loose: true,
    configCode: 'auto.detect#1313788472'
  },
  packageJSON: {
    dependencies: {
      '@angular/animations': '12.0.4',
      '@angular/cdk': '12.0.4',
      '@angular/common': '12.0.4',
      '@angular/core': '12.0.4',
      '@angular/flex-layout': '12.0.0-beta.34',
      '@angular/forms': '12.0.4',
      '@angular/material': '12.0.4',
      '@angular/platform-browser': '12.0.4',
      '@angular/platform-browser-dynamic': '12.0.4',
      '@angular/router': '12.0.4',
      '@capacitor/android': '3.0.1',
      '@capacitor/app': '1.0.1',
      '@capacitor/core': '3.0.1',
      '@capacitor/haptics': '1.0.1',
      '@capacitor/ios': '3.0.1',
      '@capacitor/keyboard': '1.0.1',
      '@capacitor/status-bar': '1.0.1',
      '@compodoc/compodoc': '^1.1.18',
      '@fortawesome/angular-fontawesome': '0.9.0',
      '@fortawesome/fontawesome-svg-core': '1.2.35',
      '@fortawesome/free-solid-svg-icons': '5.15.3',
      '@ionic-native/android-permissions': '5.33.1',
      '@ionic-native/app-version': '^5.36.0',
      '@ionic-native/core': '5.33.1',
      '@ionic-native/device': '5.33.1',
      '@ionic-native/diagnostic': '5.33.1',
      '@ionic-native/network': '5.35.0',
      '@ionic-native/screen-orientation': '5.34.0',
      '@ionic-native/sqlite': '5.33.1',
      '@ionic/angular': '5.5.2',
      '@ionic/storage-angular': '3.0.6',
      'cordova-android': '8.1.0',
      'cordova-plugin-android-permissions': '1.1.2',
      'cordova-plugin-app-version': '^0.1.12',
      'cordova-plugin-device': '2.0.3',
      'cordova-plugin-network-information': '3.0.0',
      'cordova-plugin-screen-orientation': '3.0.2',
      'cordova-plugin-sqlite-2': '1.0.7',
      'crypto-js': '^4.1.1',
      'es6-promise-plugin': '4.2.2',
      'karma-junit-reporter': '^2.0.1',
      luxon: '1.27.0',
      'object-hash': '2.2.0',
      pouchdb: '7.2.2',
      'pouchdb-adapter-cordova-sqlite': '2.0.8',
      'pouchdb-authentication': '1.1.3',
      'pouchdb-find': '7.2.2',
      rxjs: '6.6.0',
      tslib: '2.2.0',
      uuid: '3.4.0',
      'zone.js': '0.11.4'
    },
    devDependencies: {
      '@angular-devkit/build-angular': '12.0.4',
      '@angular-eslint/builder': '12.1.0',
      '@angular-eslint/eslint-plugin': '12.1.0',
      '@angular-eslint/eslint-plugin-template': '12.1.0',
      '@angular-eslint/template-parser': '12.1.0',
      '@angular/cli': '12.0.4',
      '@angular/compiler': '12.0.4',
      '@angular/compiler-cli': '12.0.4',
      '@angular/language-service': '12.0.4',
      '@capacitor/cli': '3.0.1',
      '@ionic/angular-toolkit': '4.0.0',
      '@types/jasmine': '3.6.0',
      '@types/jasminewd2': '2.0.9',
      '@types/luxon': '1.27.0',
      '@types/node': '12.11.1',
      '@types/object-hash': '2.1.1',
      '@types/pouchdb': '6.4.0',
      '@typescript-eslint/eslint-plugin': '4.16.1',
      '@typescript-eslint/parser': '4.16.1',
      'cordova-plugin-ionic-webview': '4.2.1',
      'cordova-plugin-splashscreen': '5.0.2',
      'cordova-plugin-statusbar': '2.4.2',
      'cordova-plugin-whitelist': '1.3.3',
      eslint: '7.6.0',
      'eslint-plugin-import': '2.22.1',
      'eslint-plugin-jsdoc': '30.7.6',
      'eslint-plugin-prefer-arrow': '1.2.2',
      'jasmine-core': '4.0.0',
      'jasmine-spec-reporter': '7.0.0',
      karma: '6.3.16',
      'karma-chrome-launcher': '3.1.0',
      'karma-coverage': '2.2.0',
      'karma-coverage-istanbul-reporter': '3.0.3',
      'karma-jasmine': '4.0.1',
      'karma-jasmine-html-reporter': '1.7.0',
      mermaid: '8.12.1',
      'ng-mocks': '^13.0.3',
      protractor: '7.0.0',
      'ts-node': '8.3.0',
      typescript: '4.2.4'
    }
  },
  fs: { numberOfFiles: 238 },
  debug: [
    '2022-02-25T11:39:36.641Z angular/cli config Detected Angular CLI.\n',
    '2022-02-25T11:39:36.646Z angular/cli config Angular currentDirPath: "C:\\Code\\NORA\\data-collection-app", workspaceDirPath: "C:\\Code\\NORA\\data-collection-app"\n',
    '2022-02-25T11:39:46.527Z angular/cli karma project config Using virtual C:/Code/NORA/data-collection-app/src/test.ts.\n',
    '2022-02-25T11:39:46.586Z project Wallaby Node version: v14.18.1\n',
    '2022-02-25T11:39:46.586Z project Wallaby config: C:\\Code\\NORA\\data-collection-app\\auto.detect\n',
    '2022-02-25T11:39:47.467Z project File cache: C:\\Users\\Tom.robinson\\AppData\\Local\\JetBrains\\Rider2021.3\\wallaby\\projects\\8eb8fcf4c6c6565c\n',
    '2022-02-25T11:39:47.799Z uiService Listening port 51235\n',
    '2022-02-25T11:39:47.896Z workers Parallelism for initial run: 14, for regular run: 7\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #0\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #1\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #2\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #3\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #4\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #5\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #6\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #7\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #8\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #9\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #10\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #11\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #12\n',
    '2022-02-25T11:39:47.897Z workers Starting run worker instance #13\n',
    '2022-02-25T11:39:47.900Z workers Web server is listening at 59751\n',
    '2022-02-25T11:39:47.913Z project Stopping process pool\n',
    '2022-02-25T11:39:47.913Z project File cache is up-to-date, starting full test run\n',
    '2022-02-25T11:39:47.915Z project Running postprocessor\n',
    '2022-02-25T11:39:47.921Z postprocessor Processing started.\n',
    '2022-02-25T11:39:47.921Z postprocessor Initializing.\n',
    '2022-02-25T11:39:47.922Z postprocessor Initializing entry.\n',
    '2022-02-25T11:39:47.922Z postprocessor Sanitizing configuration.\n',
    '2022-02-25T11:39:47.922Z postprocessor Adjusting configuration.\n',
    '2022-02-25T11:39:47.927Z postprocessor Creating compiler.\n',
    '2022-02-25T11:39:48.376Z postprocessor Customizing compiler.\n',
    '2022-02-25T11:39:48.376Z postprocessor Compilation started.\n',
    '2022-02-25T11:40:08.784Z uiService UI client connected\n',
    '2022-02-25T11:40:08.784Z uiService Outgoing message ui:handshake\n',
    '2022-02-25T11:40:08.787Z uiService UI client connected\n',
    '2022-02-25T11:40:08.787Z uiService Outgoing message ui:handshake\n',
    '2022-02-25T11:40:08.801Z uiService UI client disconnected\n',
    '2022-02-25T11:40:08.819Z uiService Incoming message ui:tests:resultsRequested\n',
    '2022-02-25T11:40:08.821Z uiService Outgoing message ui:tests:allResultsUpdated\n',
    '2022-02-25T11:40:08.826Z uiService Incoming message ui:start\n',
    '2022-02-25T11:40:08.828Z uiService Outgoing message ui:summary\n',
    '2022-02-25T11:40:08.830Z uiService Outgoing message ui:files\n',
    '2022-02-25T11:40:09.453Z workers Started run worker instance (immediate) #0\n',
    '2022-02-25T11:40:09.453Z workers Started run worker instance (immediate) #1\n',
    '2022-02-25T11:40:09.454Z workers Started run worker instance (immediate) #2\n',
    '2022-02-25T11:40:09.454Z workers Started run worker instance (immediate) #3\n',
    '2022-02-25T11:40:09.454Z workers Started run worker instance (immediate) #4\n',
    '2022-02-25T11:40:09.454Z workers Started run worker instance (immediate) #5\n',
    '2022-02-25T11:40:09.454Z workers Started run worker instance (immediate) #6\n',
    '2022-02-25T11:40:09.454Z workers Started run worker instance (immediate) #7\n',
    '2022-02-25T11:40:09.454Z workers Started run worker instance (immediate) #8\n',
    '2022-02-25T11:40:09.454Z workers Started run worker instance (immediate) #9\n',
    '2022-02-25T11:40:09.454Z workers Started run worker instance (immediate) #10\n',
    '2022-02-25T11:40:09.454Z workers Started run worker instance (immediate) #11\n',
    '2022-02-25T11:40:09.454Z workers Started run worker instance (immediate) #12\n',
    '2022-02-25T11:40:09.454Z workers Started run worker instance (immediate) #13\n',
    '2022-02-25T11:40:19.744Z postprocessor Compilation finished.\n',
    '2022-02-25T11:40:21.918Z postprocessor Emitting 750 files.\n',
    '2022-02-25T11:40:21.944Z postprocessor Processing finished.\n',
    '2022-02-25T11:40:22.164Z project Postprocessor execution finished\n',
    '2022-02-25T11:40:22.164Z project Test run started; run priority: 3\n',
    '2022-02-25T11:40:22.171Z project Running all tests\n',
    '2022-02-25T11:40:22.196Z workers Starting test run, priority: 3\n',
    '2022-02-25T11:40:22.196Z workers Distributing tests between 14 workers\n',
    '2022-02-25T11:40:22.198Z workers Running tests in parallel\n',
    '2022-02-25T11:40:22.199Z chromeRunner Starting sandbox [worker #0, session #1a46a]\n',
    '2022-02-25T11:40:22.199Z chromeRunner Starting sandbox [worker #1, session #06u4n]\n',
    '2022-02-25T11:40:22.199Z chromeRunner Starting sandbox [worker #2, session #oyid6]\n',
    '2022-02-25T11:40:22.199Z chromeRunner Starting sandbox [worker #3, session #5omqj]\n',
    '2022-02-25T11:40:22.199Z chromeRunner Preparing sandbox [worker #0, session #1a46a]\n',
    '2022-02-25T11:40:22.202Z browserRunner Total files to load in sandbox: 5\n',
    '2022-02-25T11:40:22.204Z browserRunner Sandbox is generated [worker #0, session #1a46a]: http://localhost:59751/wallaby_sandbox0.html\n',
    '2022-02-25T11:40:22.204Z chromeRunner Preparing sandbox [worker #1, session #06u4n]\n',
    '2022-02-25T11:40:22.205Z browserRunner Total files to load in sandbox: 5\n',
    '2022-02-25T11:40:22.207Z browserRunner Sandbox is generated [worker #1, session #06u4n]: http://localhost:59751/wallaby_sandbox1.html\n',
    '2022-02-25T11:40:22.207Z chromeRunner Preparing sandbox [worker #2, session #oyid6]\n',
    '2022-02-25T11:40:22.208Z browserRunner Total files to load in sandbox: 5\n',
    '2022-02-25T11:40:22.210Z browserRunner Sandbox is generated [worker #2, session #oyid6]: http://localhost:59751/wallaby_sandbox2.html\n',
    '2022-02-25T11:40:22.210Z chromeRunner Preparing sandbox [worker #3, session #5omqj]\n',
    '2022-02-25T11:40:22.212Z browserRunner Total files to load in sandbox: 5\n',
    '2022-02-25T11:40:22.214Z browserRunner Sandbox is generated [worker #3, session #5omqj]: http://localhost:59751/wallaby_sandbox3.html\n',
    '2022-02-25T11:40:22.214Z chromeRunner Prepared sandbox [worker #0, session #1a46a]\n',
    '2022-02-25T11:40:22.214Z chromeRunner Prepared sandbox [worker #1, session #06u4n]\n',
    '2022-02-25T11:40:22.214Z chromeRunner Prepared sandbox [worker #2, session #oyid6]\n',
    '2022-02-25T11:40:22.214Z chromeRunner Prepared sandbox [worker #3, session #5omqj]\n',
    '2022-02-25T11:40:22.215Z workers [worker #0, session #1a46a] Running tests in sandbox\n',
    '2022-02-25T11:40:22.216Z workers [worker #1, session #06u4n] Running tests in sandbox\n',
    '2022-02-25T11:40:22.216Z workers [worker #2, session #oyid6] Running tests in sandbox\n',
    '2022-02-25T11:40:22.217Z workers [worker #3, session #5omqj] Running tests in sandbox\n',
    '2022-02-25T11:40:23.960Z workers [oyid6] Loaded 18 test(s)\n',
    '2022-02-25T11:40:23.961Z workers [1a46a] Loaded 1 test(s)\n',
    '2022-02-25T11:40:23.975Z workers [1a46a] Test executed: Should return a flat list of locations only, given streets with locations as children\n',
    '2022-02-25T11:40:23.976Z workers [oyid6] Test executed: Should error if location data value is not a number\n',
    '2022-02-25T11:40:23.992Z workers [oyid6] Test executed: Should error if location data value is null\n',
    '2022-02-25T11:40:24.005Z workers [oyid6] Test executed: Should exclude on occupierCount range match when location value is too high\n',
    '2022-02-25T11:40:24.022Z workers [oyid6] Test executed: Should error if filter type is not recognised\n',
    '2022-02-25T11:40:24.040Z workers [oyid6] Test executed: Should include on subCategoryId match\n',
    '2022-02-25T11:40:24.055Z workers [oyid6] Test executed: Should error if location does not have data to filter against\n',
    '2022-02-25T11:40:24.063Z workers [1a46a] Run 1 test(s), skipped 0 test(s)\n',
    '2022-02-25T11:40:24.066Z workers [1a46a] Sandbox is responsive, closing it\n',
    '2022-02-25T11:40:24.073Z workers [oyid6] Test executed: Should exclude on boolean match\n',
    '2022-02-25T11:40:24.088Z workers [oyid6] Test executed: Should include on boolean match\n',
    '2022-02-25T11:40:24.106Z workers [oyid6] Test executed: Should error if numeric filter value is not a number\n',
    '2022-02-25T11:40:24.121Z workers [oyid6] Test executed: Should include if all include filters match\n',
    '2022-02-25T11:40:24.126Z workers [5omqj] Loaded 5 test(s)\n',
    '2022-02-25T11:40:24.137Z workers [oyid6] Test executed: Should error if numeric filter value is null\n',
    '2022-02-25T11:40:24.154Z workers [oyid6] Test executed: Should exclude on subCategoryId match, with multiple filters\n',
    '2022-02-25T11:40:24.171Z workers [oyid6] Test executed: Should exclude on subCategoryId match\n',
    '2022-02-25T11:40:24.188Z workers [oyid6] Test executed: Should include on subCategoryId match, with multiple filters\n',
    '2022-02-25T11:40:24.195Z workers [06u4n] Loaded 5 test(s)\n',
    '2022-02-25T11:40:24.203Z workers [oyid6] Test executed: Should exclude on occupierCount range match\n',
    '2022-02-25T11:40:24.220Z workers [oyid6] Test executed: Should not include on occupierCount range match when location is not valid\n',
    '2022-02-25T11:40:24.236Z workers [oyid6] Test executed: Should include on occupierCount range match when location is valid\n',
    '2022-02-25T11:40:24.267Z workers [oyid6] Test executed: Should error if field is used as an exclude and include\n',
    '2022-02-25T11:40:24.340Z workers [oyid6] Run 18 test(s), skipped 0 test(s)\n',
    '2022-02-25T11:40:24.343Z workers [oyid6] Sandbox is responsive, closing it\n',
    '2022-02-25T11:40:24.855Z workers [5omqj] Test executed: should be creatable\n',
    '2022-02-25T11:40:24.935Z workers [06u4n] Test executed: Should return null when there is not a previous location\n',
    '2022-02-25T11:40:25.312Z workers [06u4n] Test executed: Should return previous location when there is one\n',
    '2022-02-25T11:40:25.320Z workers [5omqj] Test executed: Returns 2 when there is an abandoned survey and a completed survey across two different surveys\n',
    '2022-02-25T11:40:25.603Z workers [5omqj] Test executed: Returns 1 when there is an abandoned survey\n',
    '2022-02-25T11:40:25.606Z workers [06u4n] Test executed: should be creatable\n',
    '2022-02-25T11:40:25.887Z workers [5omqj] Test executed: Returns 1 when there is an answered survey\n',
    '2022-02-25T11:40:25.890Z workers [06u4n] Test executed: Should return null when there is not a next location\n',
    '2022-02-25T11:40:26.130Z workers [5omqj] Test executed: Returns zero when there are no answered surveys\n',
    '2022-02-25T11:40:26.136Z workers [06u4n] Test executed: Should return next location when there is one\n',
    '2022-02-25T11:40:26.212Z workers [5omqj] Run 5 test(s), skipped 0 test(s)\n',
    '2022-02-25T11:40:26.214Z workers [06u4n] Run 5 test(s), skipped 0 test(s)\n',
    '2022-02-25T11:40:26.215Z workers [5omqj] Sandbox is responsive, closing it\n',
    '2022-02-25T11:40:26.218Z workers [06u4n] Sandbox is responsive, closing it\n',
    '2022-02-25T11:40:26.220Z workers Merging parallel test run results\n',
    '2022-02-25T11:40:26.233Z project Test run finished\n',
    '2022-02-25T11:40:26.235Z project Processed console.log entries\n',
    '2022-02-25T11:40:26.235Z project Processed loading sequences\n',
    '2022-02-25T11:40:26.237Z project Processed executed tests\n',
    '2022-02-25T11:40:26.258Z project Processed code coverage\n',
    '2022-02-25T11:40:26.365Z project Test run result processed and sent to IDE\n',
    '2022-02-25T11:40:26.385Z uiService Outgoing message ui:summary\n',
    '2022-02-25T11:40:26.389Z uiService Outgoing message ui:tests:allResultsUpdated\n',
    '2022-02-25T11:40:26.391Z uiService Outgoing message ui:coverageChanged\n',
    '2022-02-25T11:40:40.710Z project Requested to run some tests\n',
    '2022-02-25T11:40:40.710Z project Debugger start test run\n',
    '2022-02-25T11:40:40.711Z uiService Outgoing message ui:busy\n',
    '2022-02-25T11:40:40.712Z project Test run started; run priority: 1\n',
    '2022-02-25T11:40:40.712Z testTask Test files from affected: 0, from deleted or manually requested: 1, from recently changed: 0, from loaded by: 0, from failing: 0\n',
    '2022-02-25T11:40:40.725Z workers Starting test run, priority: 1\n',
    '2022-02-25T11:40:40.726Z chromeRunner Starting sandbox [worker #0, session #e9h4c]\n',
    '2022-02-25T11:40:40.726Z chromeRunner Preparing sandbox [worker #0, session #e9h4c]\n',
    '2022-02-25T11:40:40.728Z browserRunner Total files to load in sandbox: 5\n',
    '2022-02-25T11:40:40.729Z browserRunner Sandbox is generated [worker #0, session #e9h4c]: http://localhost:59751/wallaby_sandbox0.html\n',
    '2022-02-25T11:40:40.729Z chromeRunner Prepared sandbox [worker #0, session #e9h4c]\n',
    '2022-02-25T11:40:40.729Z workers [worker #0, session #e9h4c] Running tests in sandbox\n',
    '2022-02-25T11:40:41.922Z workers [e9h4c] Loaded 5 test(s)\n',
    '2022-02-25T11:40:42.663Z workers [e9h4c] Test executed: Returns 1 when there is an abandoned survey\n',
    '2022-02-25T11:40:42.821Z workers [e9h4c] Run 1 test(s), skipped 4 test(s)\n',
    '2022-02-25T11:40:42.824Z workers [e9h4c] Sandbox is responsive, closing it\n',
    '2022-02-25T11:40:42.828Z project Test run finished\n',
    '2022-02-25T11:40:42.828Z project Processed console.log entries\n',
    '2022-02-25T11:40:42.828Z project Processed loading sequences\n',
    '2022-02-25T11:40:42.828Z project Processed executed tests\n',
    '2022-02-25T11:40:42.845Z project Processed code coverage\n',
    '2022-02-25T11:40:42.846Z project Processed code trace\n',
    '2022-02-25T11:40:42.935Z project Test run result processed and sent to IDE\n',
    '2022-02-25T11:40:42.951Z uiService Outgoing message ui:summary\n',
    '2022-02-25T11:40:42.953Z uiService Outgoing message ui:testFilterChanged\n',
    '2022-02-25T11:40:42.960Z uiService Outgoing message ui:coverageChanged\n',
    '2022-02-25T11:40:42.969Z uiService Outgoing message ui:summary\n',
    '2022-02-25T11:40:42.970Z uiService Outgoing message ui:tests:someResultsUpdated\n',
    '2022-02-25T11:41:02.272Z project Requested to run some tests\n',
    '2022-02-25T11:41:02.272Z project Evaluated expression test run\n',
    '2022-02-25T11:41:02.272Z uiService Outgoing message ui:busy\n',
    '2022-02-25T11:41:02.273Z project Test run started; run priority: 1\n',
    '2022-02-25T11:41:02.285Z workers Starting test run, priority: 1\n',
    '2022-02-25T11:41:02.286Z chromeRunner Starting sandbox [worker #0, session #2ik1n]\n',
    '2022-02-25T11:41:02.286Z chromeRunner Preparing sandbox [worker #0, session #2ik1n]\n',
    '2022-02-25T11:41:02.287Z browserRunner Total files to load in sandbox: 5\n',
    '2022-02-25T11:41:02.288Z browserRunner Sandbox is generated [worker #0, session #2ik1n]: http://localhost:59751/wallaby_sandbox0.html\n',
    '2022-02-25T11:41:02.288Z chromeRunner Prepared sandbox [worker #0, session #2ik1n]\n',
    '2022-02-25T11:41:02.288Z workers [worker #0, session #2ik1n] Running tests in sandbox\n',
    '2022-02-25T11:41:02.839Z workers [2ik1n] Loaded 5 test(s)\n',
    '2022-02-25T11:41:03.400Z workers [2ik1n] Test executed: Returns 1 when there is an abandoned survey\n',
    '2022-02-25T11:41:03.517Z workers [2ik1n] Run 1 test(s), skipped 4 test(s)\n',
    '2022-02-25T11:41:03.518Z workers [2ik1n] Sandbox is responsive, closing it\n',
    '2022-02-25T11:41:03.522Z project Test run finished\n',
    '2022-02-25T11:41:03.522Z project Processed console.log entries\n',
    '2022-02-25T11:41:03.522Z project Processed loading sequences\n',
    '2022-02-25T11:41:03.522Z project Processed executed tests\n',
    '2022-02-25T11:41:03.532Z project Processed code coverage\n',
    '2022-02-25T11:41:03.532Z project Processed code trace\n',
    '2022-02-25T11:41:03.547Z project Test run result processed and sent to IDE\n',
    '2022-02-25T11:41:03.556Z uiService Outgoing message ui:summary\n',
    '2022-02-25T11:41:03.557Z uiService Outgoing message ui:tests:someResultsUpdated\n',
    '2022-02-25T11:45:11.969Z project Debugger stopped\n',
    '2022-02-25T11:45:11.971Z uiService Outgoing message ui:summary\n',
    '2022-02-25T11:45:11.971Z uiService Outgoing message ui:testFilterChanged\n',
    '2022-02-25T11:45:11.973Z uiService Outgoing message ui:coverageChanged\n'
  ]
}
```
</details>

**Comment by ArtemGovorov** [comment]
Thanks for reporting the issue. The problem was Rider specific only. It's now fixed and the fix is published in the latest plugin v1.0.233.

**Comment by tjrobinson** [comment]
@ArtemGovorov Thank you for the speedy fix, it works great now.
